server/Service/Services/ResetPasswordService.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Service.Interfaces;
+using Services.Models;
+using WebAPI.Services;
+
+namespace Services.ResetPasswordService
+{
+  public class ResetPasswordService
```

Comment:
what is the purpose of this class? does it do anything usefull except instantiating?
or is it just duplicate of another service down the line?

Reply:
strange, it shouldn't have happened. I deleted it from vs, but it was surprisingly left in the file system. now deleted